doc/how-to/ceph_networking.rst

```diff
@@ -0,0 +1,155 @@
+        lxc config device add micro3 eth2 nic network=cephbr0 name=eth2
+        lxc config device add micro4 eth2 nic network=cephbr0 name=eth2
+        lxc config device add micro1 eth3 nic network=cephbr1 name=eth3
+        lxc config device add micro2 eth3 nic network=cephbr1 name=eth3
+        lxc config device add micro3 eth3 nic network=cephbr1 name=eth3
+        lxc config device add micro4 eth3 nic network=cephbr1 name=eth3
+
+3. Now, just like in the tutorial, start the VMs.
+
+4. On each VM, bring the network interfaces up and give them an IP address within their network subnet:
+    a. For the ``cephbr0`` network, do the following for each VM::
+
+        # If the `cephbr0` gateway address is `10.0.1.1/24` (subnet should be `10.0.1.0/24`)
+        ip link set enp7s0 up
+        # `X` should be a number between 2 and 254, different for each VM
+        ip addr add 10.0.1.X/24 dev enp7s0
+
+    b. Do the same for ``cephbr1`` on each VM::
+
+        # If the `cephbr1` gateway address is `10.0.2.1/24` (subnet should be `10.0.2.0/24`)
+        ip link set enp8s0 up
+        # `X` should be a number between 2 and 254, different for each VM
+        ip addr add 10.0.2.X/24 dev enp8s0
+
+5. Now, you can start the MicroCloud initialisation process and provide the subnets you noted down in step 1.c when asked for the Ceph networking subnets.
+
+#. We will use ``cephbr0`` for the Ceph internal traffic and ``cephbr1`` for the Ceph public traffic. In the real life, you'd choose the fast subnet for the internal Ceph traffic::
+
+    What subnet (either IPv4 or IPv6 CIDR notation) would you like your Ceph public traffic on? [default: 203.0.113.0/24]: 10.0.2.0/24
+
+    Interface "enp7s0" ("10.0.2.3") detected on cluster member "micro2"
+    Interface "enp7s0" ("10.0.2.4") detected on cluster member "micro3"
+    Interface "enp7s0" ("10.0.2.2") detected on cluster member "micro1"
+
+    What subnet (either IPv4 or IPv6 CIDR notation) would you like your Ceph internal traffic on? [default: 203.0.113.0/24]: 10.0.1.0/24
+
+    Interface "enp7s0" ("10.0.1.3") detected on cluster member "micro2"
+    Interface "enp7s0" ("10.0.1.4") detected on cluster member "micro3"
+    Interface "enp7s0" ("10.0.1.2") detected on cluster member "micro1"
+
+7. The MicroCloud initialisation process will now continue as usual and the Ceph cluster will be configured with the networking setup you provided.
+8. You can inspect the Ceph network setup:
+    a. Inspect the Ceph configuration file:
+
+        .. terminal::
+            :input: microceph.ceph config dump
+            :user: root
+            :host: micro1
+            :scroll:
+
+            WHO     MASK  LEVEL     OPTION                       VALUE        RO
+            global        advanced  cluster_network              10.0.1.0/24  *
+            global        advanced  public_network               10.0.2.0/24  *
+            global        advanced  osd_pool_default_crush_rule  2
```

Comment:
Too much indentation here as well (this time it's visible because of the line in front of the terminal view).

And the same for the following substeps.

Reply:
fixed